plugins/logo_maker.py

```diff
@@ -0,0 +1,67 @@
+# © JigarVarma2005
+# Logo maker using brandcrowd.com
+# Moded from @JVToolsBot by @UniversalBotsUpdate
+import os
+import random
+import requests
+from bs4 import BeautifulSoup
```

Comment:
could u use aiohttp library instead of requests?

Reply:
Any issue with requests?

Reply:
I will try